common/src/network.rs

```diff
@@ -3,12 +3,37 @@
 use bitcoin::blockdata::block::{Block, BlockHeader};
 use bitcoin::consensus::params::Params;
 use bitcoin::hash_types::BlockHash;
+use bitcoin::network::constants::ServiceFlags;
 use bitcoin_hashes::hex::FromHex;
 
 use bitcoin_hashes::sha256d;
 
 use crate::block::Height;
 
+/// Peer services supported by nakamoto.
+#[derive(Debug, Copy, Clone)]
+pub enum Service {
```

Comment:
Should be plural as mentioned earlier: `Services`, just like `ServiceFlags`, because `Service::All` doesn't read correctly otherwise.